api/tests/unit/domain/usecases/remove-authentication-method_test.js

```diff
@@ -172,85 +206,54 @@ describe('Unit | UseCase | remove-authentication-method', function () {
   });
 
   context('When type is POLE_EMPLOI', function () {
-    it('should remove POLE_EMPLOI authentication method', async function () {
+    it('removes POLE_EMPLOI authentication method', async function () {
       // given
-      const type = OidcIdentityProviders.POLE_EMPLOI.code;
+      const type = "POLE_EMPLOI";
```

Comment:
Utiliser des simples quotes (il semble que notre linting ne fasse plus aussi complètement son travail) : 

```suggestion
      const type = 'POLE_EMPLOI';
```